books/RayTracingInOneWeekend.html

```diff
@@ -1842,7 +1842,8 @@
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ C++
     if (world.hit(r, interval(0.001, infinity), rec)) {
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-    [Listing [reflect-tolerance]: <kbd>[main.cc]</kbd> Calculating reflected ray origins with tolerance]
+    [Listing [reflect-tolerance]: <kbd>[main.cc]</kbd>
+    Calculating reflected ray origins with tolerance]
```

Comment:
Can/should this be indented?

Reply:
Yeah, I tried it, but honestly it all looks awkward. It's sensitive to which characters the line breaks on, and the multiple different brackets just make it look even more awkward. Flush left was the best I found.